x5e/gink
# added pagination for container entries

to test:
`make clean && make`
`npm run serve`

go to `localhost:8080`

to really see the pages, you may want to add:
```js
const globalDir = window.instance.getGlobalDirectory();
for (let i = 0; i < 50; i++) {
await globalDir.set(`test${i}`, "testing");
}
```

to dashboard.html after window.connectTo(), then refresh the page
